src/ResourceManagement/Sql/SqlManagement/Generated/ServerDisasterRecoveryConfigurationOperations.cs

```diff
@@ -671,7 +671,7 @@ public SqlManagementClient Client
                         TracingAdapter.ReceiveResponse(invocationId, httpResponse);
                     }
                     HttpStatusCode statusCode = httpResponse.StatusCode;
-                    if (statusCode != HttpStatusCode.OK && statusCode != HttpStatusCode.NoContent)
```

Comment:
apparently this makes one of the tests fail.  I'll undo the change to this file, but it was autogenerated from hydra spec.